ui/oopsyraidsy/oopsy_summary_list.ts

```diff
@@ -194,7 +194,7 @@ export class OopsySummaryList implements MistakeObserver {
     pullDiv.innerText = `Pull ${this.pullIdx}`;
     headerDiv.appendChild(pullDiv);
     const zoneDiv = document.createElement('div');
-    if (this.zoneName)
+    if (this.zoneName !== undefined && this.zoneName.length > 0)
```

Comment:
```suggestion
    if (this.zoneName !== undefined)
```

Arguably the zone name should never be empty?